include/defines.h

```diff
@@ -275,9 +275,9 @@ enum KART_AI_BEHAVIOURS {
     BEHAVIOUR_NONE = 0,
     BEHAVIOUR_1,
     BEHAVIOUR_HOP,
-    BEHAVIOUR_3,
-    BEHAVIOUR_4,
-    BEHAVIOUR_5,
+    BEHAVIOUR_GO_CENTER,
+    BEHAVIOUR_GO_INNER,
```

Comment:
BEHAVIOUR_DRIVE_CLOSE ? I couldn't find a better word for inner. other than near or close

BEHAVIOUR_DRIVE_CONGESTED ? CONSOLIDATED CROWDED

I like crowded.

Reply:
BEHAVIOUR_GO_CENTER, BEHAVIOUR_GO_INNER, BEHAVIOUR_GO_OUTER are consistent each other so maybe change GO to move

Reply:
What's wrong with drive?